### DISCUSSION TITLE
v2.1.3

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#595] [Announcements]
**Quality of Life:**
- Improved settings and context menus

**Bug Fixes:**
- Fixed a typo that broke ESync
- Standardised logs to be consistent across locales

**Full Changelog:** https://github.com/Whisky-App/Whisky/compare/v2.1.2...v2.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/Whisky-App/Whisky/releases/tag/v2.1.3'>v2.1.3</a>.</em>

**Comment by charlemagnec** [comment]
Awesome update. Since adding back in Esync D2R has been performing so much better. Previously on Act IV I'm getting really low FPS (even on moderate settings and resolution) now everything is smooooooth

**Comment by sarangKP** [comment]
Plss help im facing some issue it started when i accidently deleted some file, i tried deleting and re installing whisky still this error comes up its not showing anything here, even after i keep creating bottles nothing come up??? i waited few minutes still the created bottle is not showing anyone know what to do??
![Uploading Screenshot 2023-11-08 at 11.57.42 PM.png…]()

**Reply by sarangKP** [reply]
When i try opening the file all i see is Metadata.plist  , before it had more files like c drive etc but non of that is there now and i cant run any exe files aswell

**Reply by IsaacMarovitz** [reply]
Hi there, the screenshot you sent didn't upload correctly, but I'd recommend deleting the `Whisky` folder in `~/Library/Containers/`

**Reply by sarangKP** [reply]
Thank you for that info, i tried it now and it worked, and ig the problem earlier was that i didnt realise the file was 40 gb or so, so all i had to do was wait a while, my bad 😥

**Comment by LabOfCuriosity** [comment]
![Screenshot 2023-11-18 at 10 20 49 AM](https://github.com/Whisky-App/Whisky/assets/151392068/0693019b-b35a-4188-a224-af37e326d152)

1. Can you explain why when I installed Whisky on an M1 Mac - Sonoma 14.0, my Avast File Shield blocked a 'Win32:WormX-Gen [Wrm]'?
File path:  /User/Admin/Library/Application Support/com/isaacmaovitz.Whisky/Libraries/Wine/lib/wine/i386-windows/msxml6.dll
Process:   /usr/bin/bsdtar
Unsure of what this meant, I googled it and saw this article. https://howtofix.guide/win32wormx-gen-wrm/

2. Is there a process/function that requires this particular version of msxml6.dll with whatever got flagged to work?
2.b. Is there a different, verified version of the msxml6.dll that can be swapped in and work? https://malwaretips.com/blogs/msxml6-dll-what-it-is-how-to-fix-errors/ 

3. I'm curious if that version of msxml6.dll is what was packaged with Wine, or if it was something you customized to get your wrapper to work or something. I prefer asking and knowing stuff instead of guessing. 

Learning abut this stuff is really cool! And thanks for making this process so simple for Apple Silicon!

Thanks!

**Reply by IsaacMarovitz** [reply]
This is a known false positive. Antivirus software on Mac is disruptive and unnecessary.

Full details [here](https://github.com/Whisky-App/Whisky/issues/465).

**Comment by zTironz** [comment]
Anyone have problems with old AMD drivers? I run cs2 and Hogwarts with whisky and have this issue

**Comment by zhangBlack** [comment]
I would like to ask, after installing Steam, and then starting Diablo 4 in Steam, the game cannot run. Is there a solution to this?

**Reply by arog00** [reply]
I'm having the same issue, there hasn't been any progress on this in weeks
